src/app/views/query-runner/request/permissions/Permission.tsx

```diff
@@ -273,14 +282,15 @@ export class Permission extends Component<IPermissionProps, IPermissionState> {
   }
 }
 
-function mapStateToProps({ sampleQuery, scopes, authToken, consentedScopes, dimensions, permissionsPanelOpen }: IRootState) {
+function mapStateToProps({ sampleQuery, scopes, authToken, consentedScopes, dimensions, permissionsPanelOpen, permissionModeType }: IRootState) {
```

Comment:
This line is getting a tiny bit long, in my opinion; I am actually unfamiliar with TS/JS checkstyle commonalities for functions so I don't know how to split it personally

Reply:
Could be something like 
```suggestion
function mapStateToProps({ 
    sampleQuery, 
    scopes, 
    authToken, 
    consentedScopes, 
    dimensions, 
    permissionsPanelOpen, 
    permissionModeType }: IRootState) {
```

Reply:
Thanks for the suggestion! I think that looks a lot better 😄

Reply:
Resolving in #23 - Review when you can!